Eatery/Views/Eateries/CampusEateryStationTableHeaderView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CampusEateryStationTableHeaderView.swift
+//  Eatery
+//
+//  Created by Ethan Fine on 2/27/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class CampusEateryStationTableHeaderView: UITableViewHeaderFooterView {
+    
+    private var headerView: UIView
+    private var sectionLabel: UILabel
```

Comment:
can we make these `private let` and initialize them here

Reply:
On bump, there is no actual extra whitespace there